src/pages/bug-bounty/vulnerabilities-types/index.tsx

```diff
@@ -90,7 +90,7 @@ const VulnerabilitiesTypes = () => {
                                 <Flex mt="40px" ai="center">
                                     <IconWrap src={item.icon} />
                                     <Header as="h2" type="subtitle-1" ml="8px">
-                                        {localize(item.title)}
```

Comment:
I think we need to translate the title here, don't we?

Reply:
The `item.title` already has it's own `<Localized />` component in the `vulnerability_types_content` array.